packages/calling-stateful-client/src/StatefulCallClient.ts

```diff
@@ -157,6 +157,20 @@ export interface StatefulCallClient extends CallClient {
    * @public
    */
   createCallAgent(...args: Parameters<CallClient['createCallAgent']>): Promise<DeclarativeCallAgent>;
+
```

Comment:
Just a bit curious about this and also the sentence above - is this breaking the behavior of declarative concept?
 Let's say there is a function previously can accept normal calling sdk client:

doSomething(callClient: CallClient)

Is it still acceptable that it is called like this?
doSomething(statefulCallClient)

Reply:
So I added this because the SDK has this function on their CallClient to make the TeamsCallAgent. I would think this helps us declaritify the SDK callClient since we were missing this function. I could be wrong here though

Reply:
To test, can we pass in a statefulClient with these changes into a function expecting a callClient